src/typegenPrinter.ts

```diff
@@ -159,13 +160,26 @@ export class TypegenPrinter {
       }
     }
 
-    eachObj(rootTypings, (val, key) => {
-      if (typeof val !== 'string') {
-        const importPath = (path.isAbsolute(val.path) ? relativePathTo(val.path, outputPath) : val.path)
+    eachObj(rootTypings, (rootType, typeName) => {
+      if (typeof rootType !== 'string') {
+        const rootTypePath = rootType.path
+
+        if (typeof rootTypePath !== 'string' || !path.isAbsolute(rootTypePath)) {
+          throw new Error(
+            `Expected an absolute path for the root typing path of the type ${typeName}, saw ${rootTypePath}`
+          )
+        }
+
+        if (!fs.existsSync(rootTypePath)) {
+          throw new Error(`Root typing path ${rootTypePath} of the type ${typeName} does not exist`)
+        }
+
+        const importPath = relativePathTo(rootType.path, outputPath)
           .replace(/(\.d)?\.ts/, '')
           .replace(/\\+/g, '/')
+
         importMap[importPath] = importMap[importPath] || new Set()
-        importMap[importPath].add(val.alias ? `${val.name} as ${val.alias}` : val.name)
+        importMap[importPath].add(rootType.alias ? `${rootType.name} as ${rootType.alias}` : rootType.name)
```

Comment:
This is technically a breaking change as I'm now only accepting absolute file paths, to ensure the ability to check whether the file exists.

@tgriesser I'm not sure what was the use-case for enabling any kind of path. Path aliases maybe? Do you feel strongly about keeping any string allowed? In that case, I'll make the checks only in case the path is absolute